### DISCUSSION TITLE
Hey does this library only work for .xyz domain endings?

### DISCUSSION THREAD

**Discussion by davidsmooke** [devdotxyz/eth-xyz#66] [Q&A] [answered]
hey does this method of displaying nfts only work for .xyz domain endings?

**Comment by jhxyz** [comment, marked as answer]
Hello @davidsmooke sorry for the astronomical delay, this works for any name in ENS! (ens.domains)
That means it supports any .eth name, and any domain name claimed in ENS.

For example: cooperray.eth -> [cooperray.eth.xyz](https://cooperray.eth.xyz)

To view the profile of any domain name claimed in ENS, you can go to the profile like this:
https://eth.xyz/ `domain name`
i.e. https://eth.xyz/walletname.xyz

This works for all domain names that you can claim in ENS:
https://eth.xyz/decashed.com
